server/api/v1/pet/pet.router.test.js

```diff
@@ -0,0 +1,4 @@
+const supertest = require('supertest');
```

Comment:
Please change this file name to be pet.test.js

Reply:
Done